### DISCUSSION TITLE
邮件中链接403

### DISCUSSION THREAD

**Discussion by Guyao146** [bs-community/blessing-skin-server#418] [Q&A] [answered]
### Blessing Skin 版本

v6.0.0-rc.1

### PHP 版本

8.0

### 出现问题时所使用的浏览器

Microsoft Edge

### 您正在使用的 Web Server

Apache

### 您正在使用宝塔吗？

- [x] 是

### 出现了什么问题？

hiii 我想问下邮件发送成功了，但是邮件里的链接一点就403是咋回事啊
期望的行为：正常验证
宝塔 防跨站已关闭 php禁用函数已全部解除禁用 常用拓展fileinfo exif已安装
除了点击皮肤站发送的邮件是403之外其他都是完全正常使用的

### 错误日志

```text
https://paste.gg/p/anonymous/ffe293211da64643b82a521fe31609fd
```

### 重现步骤

1.迁移皮肤站到另一个服务器
2.用户注册皮肤站发送邮件点击邮件中的链接出现403错误
![QQ截图20220616192553](https://user-images.githubusercontent.com/61780021/174060104-dc0c23a5-59cf-41e1-b2ac-f6c81004df88.png)

**Comment by mochaaP** [comment, marked as answer]
建議檢查一下是不是已經通過驗證了，有可能反釣魚自動幫你點了。
